docs/time_series_causal_graph.md

```diff
@@ -321,10 +327,11 @@ ts_cg: TimeSeriesCausalGraph
 backward_steps = 2
 forward_steps = 3
 
+# With all parents
 extended_graph = ts_cg.extend_graph(backward_steps=backward_steps, forward_steps=forward_steps)
 
-# With all parents
-extended_graph = ts_cg.extend_graph(backward_steps=backward_steps, forward_steps=forward_steps, include_all_parents=True)
+# Without all parents
+extended_graph = ts_cg.extend_graph(backward_steps=backward_steps, forward_steps=forward_steps, include_all_parents=False)
 ```
 
```

Comment:
I can't comment on the line but need to update the `## Example` section below.  It currently has
```
If you extend the minimal graph `backward_steps=2` and `forward_steps=0` with default arguments:

![ts_cg](images/ts_extended_graph.png)

Finally, if you extend the minimal graph with `backward_steps=2` and `forward_steps=0` but setting 
`include_all_parents=True`:

![ts_cg](images/ts_extended_graph_all_parents.png)
```
So just need to flip it essentially. Can see what it currently looks like [here](https://causalgraph.causalens.com/docs/generated/cai-causal-graph/docs/time_series_causal_graph#example).